.asf.yaml

```diff
@@ -27,7 +27,7 @@ notifications:
   jira_options: link label worklog
 github:
   description: "Apache DataFusion SQL Query Engine"
-  homepage: https://arrow.apache.org/datafusion
+  homepage: https://datafusion.apache.org/
```

Comment:
It should be `https://datafusion.apache.org/datafusion/`.

Reply:
`https://datafusion.apache.org/` will give you:

![Screenshot 2024-04-22 at 8 19 35 PM](https://github.com/apache/datafusion/assets/68855/7b9c7edb-4b69-40b7-bfcc-57ae979b0d35)

Reply:
After this change it should start publishing to https://datafusion.apache.org without needing the subdirectory 🤞